library/Translate/src/Juvix/Core/Common/Context/Traverse.hs

```diff
@@ -0,0 +1,119 @@
+{-# LANGUAGE OverloadedLists #-}
+
+-- | Calculate mutually-recursive groups of definitions.
+module Juvix.Core.Common.Context.Traverse
+  ( traverseContext,
+    traverseContext1,
+    traverseContext_,
+    traverseContext1_,
+    Entry (..),
+    Group,
+    Groups,
+    recGroups,
+  )
+where
+
+import qualified Data.Graph as Graph
+import qualified Data.HashSet as HashSet
+import qualified Generics.SYB as SYB
+import Juvix.Core.Common.Context.Traverse.Types
```

Comment:
I found this dep a bit hard to figure out what was in it, given how it has more functions than just run, env, etc.

(though this could just be because my jump to source isn't online while reviewing this yet, as I didn't recompile things)

Reply:
I can move the functions out to the main `Traverse` module if that would be clearer, or rename `Types` to something else?